services/brig/src/Brig/API/Internal.hs

```diff
@@ -105,12 +106,14 @@ import Wire.API.User.RichInfo
 -- Sitemap (servant)
 
 servantSitemap ::
-  Members
-    '[ BlacklistStore,
-       GalleyProvider,
-       UserPendingActivationStore p
-     ]
-    r =>
+  ( Members
+      '[ BlacklistStore,
+         GalleyProvider,
+         UserPendingActivationStore p
+       ]
+      r,
+    CallsFed 'Brig "on-user-deleted-connections"
```

Comment:
I'd expect the constraint to be gone by now, i.e., exactly at the point where endpoints are served. Has the constraint made it to here by mistake?